src/components/NewCommentForm.tsx

```diff
@@ -99,34 +94,31 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
             </span>
           )}
         </div>
-
         {errors.name && (
           <p className="help is-danger" data-cy="ErrorMessage">
             Name is required
           </p>
         )}
       </div>
 
+      {/* Email Field */}
       <div className="field" data-cy="EmailField">
         <label className="label" htmlFor="comment-author-email">
           Author Email
         </label>
-
         <div className="control has-icons-left has-icons-right">
           <input
-            type="text"
+            type="email"
```

Comment:
Ensure that the email input type is correctly handled by the form validation logic. The current setup checks for non-empty values, but additional validation for email format might be beneficial.